### DISCUSSION TITLE
How to get involved with helping

### DISCUSSION THREAD

**Discussion by fnielson1** [onimp/oni_multiplayer#5] [General]
Hey, so I've been thinking of doing this mod for years now and you are already leagues ahead of what I have ever done! 

I was wondering how I could help. I saw your roadmap and I figured I could start with the
`Printer is not synced` feature.

Also, I've used Peter Han's PLibCore library for logging and it is quite nice. Do you have anything against me including it in a pull request?

Thanks,
Ninjor (my handle on Discord).

P.S There is an excellent Discord server with a #mod-dev channel that I find to be quite useful (not my own). Here is an invite if you are interested.
https://discord.gg/oxygennotincluded

The people there are super knowledgeable and have gotten me along further than I otherwise would've. I wouldn't be surprised if you are already a member of this server.

**Comment by zuev93** [comment]
Hi Ninjor,

I would be glad to get your (and anyone else's) supports with the development.
So far I haven't created dedicated discord or other channel to communicate with contributors, however you can ping me in the discord ( bobuk#4695 ).
I also have not created any process around contributing because I don't see a need in them yet. But maybe I'll do something about it later on :)

Talking about a thing to work on:
- If you are interested in the `Printer is not synced` - feel free to take it :)
- If you see a value in the PLibCore - feel free to add it. At some point (pretty distance) I would like to spend time on optimisation (and bundle size might be one of them) but so far - no objections from my side at all.
- I'm currently working on trying to sync dupes (and creatures) behaviour
- Also it will be useful to update `Roadmap` to indicate which areas are in progress and by whom

Also thanks for the discord invite, I'll take a look on it later on :)

**Comment by fnielson1** [comment]
Excellent! Thanks!

I'm fairly new to modding so my speed will be less than optimal :)
I figured that would not be a problem.

Thanks again for letting me help!
